test/library/packages/ZMQ/req-rep.chpl

```diff
@@ -121,6 +128,15 @@ proc Worker() {
     writeln("val = ", val);
   }
 
+ // Bytes
+  {
+    var val = b"The quick \xff\xff\xff brown fox...";
+    socket.send(val);
+    val = socket.recv(bytes);
+    writeln("val = ", val);
+  }
```

Comment:
This seems useful, but I'm surprised not to see at least one test for DEALER and ROUTER

Reply:
Preferably more since, as you updated the comment, there are now five new patterns to test